asyncpg/pool.py

```diff
@@ -625,8 +627,14 @@ def create_pool(dsn=None, *,
        An :exc:`~asyncpg.exceptions.InterfaceError` will be raised on any
        attempted operation on a released connection.
     """
+    if not issubclass(connection_class, connection.Connection):
```

Comment:
We should add `Connection` to `__init__`, so that `asyncpg.Connection` is a thing.